### DISCUSSION TITLE
Why does my .NET MAUI app say that it "... would like to find and connect to devices on your local network"?

### DISCUSSION THREAD

**Discussion by rogersm-uwosh** [dotnet/maui#20257] [Q&A]
I don't understand why that message is popping up, I don't need to connect to anything local. Any ideas? If it helps, here's my .plist and list of NuGet dependencies.

Any help would be really appreciated!

![Screenshot 2024-01-30 at 8 39 15 PM](https://github.com/dotnet/maui/assets/91431120/14839d34-0850-4508-a44c-2cf4fa2ab755)
![Screenshot 2024-01-30 at 8 39 10 PM](https://github.com/dotnet/maui/assets/91431120/25f39c83-c5f7-440a-9452-33f52a807aee)

**Comment by riccardominato** [comment]
I had the same issue. It seems the popup appears only when debugging on a physical device. It's actually weird that info about this behavior can't  be easily found.

**Comment by Deathspike** [comment]
I can confirm that this is related to a recent update of MAUI or .NET. This did not happen in 8.0.0, but does happen in 8.0.4. I'm unsure what version of MAUI I had before updating my local environment. It also only happens when a debugger is attached. It does not happen otherwise. Seems relatively safe to ignore.
